src/api/cpp/cvc5_kind.h

```diff
@@ -46,6 +46,10 @@ enum Kind : int32_t
   /**
    * Internal kind.
    *
+   * This kind serves as an abstraction for internal kinds that are not exposed
+   * via the API but may appear in terms returned by API functions, e.g.,
+   * when querying model values.
```

Comment:
```suggestion
   * when querying the simplified form of a term.
```

Reply:
I actually think we don't expose internal kinds via model values?  Or if we do, it should not happen often.

The one case I can see would be `simplify` will often return terms of internal kind.  Maybe this is the better example here.